File: solutions/Python/raindrops/2/raindrops.py
sounds = ['Pling', 'Plang', 'Plong']

def convert(number):
    result = ''
    for sound in sounds:
        if number %3 ==0:
            result += sounds[0]
        if number %5 ==0:
            result += sounds[1]
        if number %7 ==0:
             result += sounds[2]
        return result or str(number)

File: solutions/C/raindrops/1/raindrops.c
#include "raindrops.h"

enum sounds {
    PLING  = 3,
    PLANG = 5,
    PLONG = 7
};

char *convert(char result[], int drops){
   if (drops % PLING == 0)
    strcat(result, "Pling");
   if (drops % PLANG == 0)
    strcat(result, "Plang");
   if (drops % PLONG == 0)
    strcat(result, "Plong");
   if (!*result)
    sprintf(result, "%d", drops);
    return result; 
}


